app/api/core/logging.rb

```diff
@@ -1,16 +1,6 @@
 # frozen_string_literal: true
 
 module Core::Logging
-  def self.logging_helper(name)
```

Comment:
Assume this is unused? Would just check the deployment project too as we have some logging setup in there.